twyla/service/queues.py

```diff
@@ -12,44 +12,36 @@
 # TODO: add actual logging
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
+def split_event_name(event_name: str):
+    assert "." in event_name, "Event names should be of format domain.event_name"
+    splat = event_name.split('.', 1)
+    return splat[0], splat[1]
```

Comment:
Since there is a `max_split` argument passed into `split`, could we just unpack line 17 to be `head, tail = event_name.split('.', 1)` and `return head, tail`? Or even just do `return event_name.split('.', 1)`?